app.js

```diff
@@ -0,0 +1,45 @@
+const booksContainer = document.getElementById('books');
+const inpTitle = document.getElementById('inp-title');
+const inpAuthor = document.getElementById('inp-author');
+
+const bookDatas = [];
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+function generateBook() {
+  booksContainer.innerHTML = '';
+  for (let i = 0; i < bookDatas.length; i += 1) {
+    booksContainer.innerHTML += `<div class="book">
+                     <div>${bookDatas[i].title}</div>
+                     <div>${bookDatas[i].author}</div>
+                     <button class="rem-btn" onclick="removeBook(${i})">Remove</button>
+                     <hr><br></div>`;
+  }
+}
+
```

Comment:
- [ ] On page reload, your app should fetch the books from the local storage and display them on the page. But that is not so. Nothing gets displayed on the page. Kindly fix that.

  ![sav-e](https://user-images.githubusercontent.com/81468675/182720091-fdcf09f2-12dc-4c2f-a552-0e9964a9e78d.jpg)